src/hardware/setup.rs

```diff
@@ -265,6 +268,8 @@ pub fn setup(
         .per_ck(design_parameters::TIMER_FREQUENCY.convert())
         .pll2_p_ck(100.MHz())
         .pll2_q_ck(100.MHz())
+        .pll3_p_ck(50.MHz())
```

Comment:
Why two outputs?

Reply:
I have to initialize the p output in order for the r output to work. Otherwise the HAL panics.

Reply:
Weird. But ok.
But do explain that.
And also add pointers/explanation of the prescaler issue in the hal (we should ultimagely use standard per_ck with a /2 additional divider, right?).

Reply:
Yes I'm still trying to figure out why the prescaler register is always reset (even if I delete the resets/restarts in the HAL)...